src/Chapter1.hs

```diff
@@ -631,9 +645,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = (lastDigit n) + secondLastDigit
```

Comment:
Nice reusage of previous `lastDigit` function! 👏🏼 

I see that you have already a var to represent this bit – `lastD`, you can reuse it here too!

```suggestion
sumLast2 n = lastD + secondLastDigit
```